### PR TITLE
CA-5164 Move CustomerCalls to V2

### PR DESCRIPTION
**Pull request by a212**

### CODE REVIEW

File: lib/lobbyist/v2/customer_call.rb [hunk 1]
```diff
@@ -1,24 +1,35 @@
 module Lobbyist
   module V2
+    class CustomerCall < Lobbyist::Base
```
Comment (oafridi) [inline lib/lobbyist/v2/customer_call.rb]: why did you change this ?

Comment (a212) [inline lib/lobbyist/v2/customer_call.rb]: Oh, thanks, I didn't change the base class name too.
I just moved CustomerCall to V2::CustomerCall, but there was already class V2::CustomerCall so I moved it to V2::CompanyCustomerCall.
